packages/box/README.md

```diff
@@ -41,10 +41,10 @@ import Box from '@leafygreen-ui/box';
 
 ## Properties
 
-| Prop        | Type                                  | Description                                                                                                                         | Default |
-| ----------- | ------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------- |
-| `component` | `HTMLElement` or `React.ReactElement` | The component or HTML tag to be rendered by the `<Box />` component. **Note**: This will supersede the behavior of any other props. | `div`   |
-| `href`      | `string`                              | When provided, `<Box />` will render an anchor tag with this `href` value.                                                          |         |
-| `children`  | `React.ReactElement`                  | Content to be rendered in an HTML element, or provided as a prop to the rendered component                                          |         |
+| Prop       | Type                                  | Description                                                                                                                         | Default |
+| ---------- | ------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| `as`       | `HTMLElement` or `React.ReactElement` | The component or HTML tag to be rendered by the `<Box />` component. **Note**: This will supersede the behavior of any other props. | `div`   |
```

Comment:
This I guess will be breaking?

Reply:
Yes indeed

Reply:
Was holding off on Changesets until we decide impact on downstream components and how to version those